src/views/virtualmachines/details/tabs/configuration/storage/components/hooks/usePVCBound.ts

```diff
@@ -0,0 +1,18 @@
+import {
```

Comment:
I think this is a good candidate for `/src/utils/hooks`.

Reply:
I am not sure because we don't need this in other places than VM's disks list.